### PR TITLE
Helicopter Gunship

### PR DESCRIPTION
**Pull request by givehub99**
Adds Helicopter Gunship to the game, with icon. Upgrades from Anti-Tank Gun

### CODE REVIEW

File: android/assets/jsons/Civ V - Vanilla/Units.json [hunk 1]
```diff
@@ -1232,9 +1232,23 @@
...
+		// Unit embarks like any other ground unit and has penalty for attacking over river or from embarked state
+		// "Unable to capture cities"
+		"name": "Helicopter Gunship",
+		"unitType": "Melee",
+		"movement": 6,
+		"strength": 60,
+		"cost": 425,
+		"requiredTech": "Computers",
+		"requiredResource": "Aluminum",
+		"uniques": ["Bonus vs Armor 100%", "No defensive terrain bonuses", "Can move after attacking", "All tiles costs 1", "Can pass through impassable tiles", "Unable to capture cities"],
```
Comment (yairm210) [inline android/assets/jsons/Civ V - Vanilla/Units.json:1249]: Change to "All tiles cost 1 movement" if we're canonizing this

Comment (givehub99) [reply android/assets/jsons/Civ V - Vanilla/Units.json:1249]: should I still allow for "All tiles cost 1" for mods that are already using it?

Comment (yairm210) [reply android/assets/jsons/Civ V - Vanilla/Units.json:1249]: ... Yeah, for now

Comment (givehub99) [reply android/assets/jsons/Civ V - Vanilla/Units.json]: Done

**Pull request merged**